src/lib/math.test.ts

```diff
@@ -1,26 +1,35 @@
 import {clamp, Vec2, AffineTransform, Rect} from './math'
-import * as jsc from 'jsverify'
+import fc from 'fast-check'
 
-test('clamp', () => {
-  jsc.assertForall(jsc.number, jsc.number, jsc.number, (a, b, c) => {
-    const result = clamp(a, b, c)
-    if (a < b) return result == b
-    if (a > c) return result == c
-    return result == a
+// Change this to fc.integer() to debug failures more easily
+// let numericType = fc.integer()
+let numericTypeMinAccuracy = 1e-3 // reduce accuracy of double generator
+let numericType = fc
+  .double({
+    next: true,
+    noDefaultInfinity: true,
+    noNaN: true,
+    min: -1 / numericTypeMinAccuracy,
+    max: 1 / numericTypeMinAccuracy,
   })
-})
+  .map(d => Math.round(d / numericTypeMinAccuracy) * numericTypeMinAccuracy)
```

Comment:
Here the arbitrary `fc.double` was able to find potential issues when used with full resolution. Contrary to `jsc.number` it can generate any number from -Infinity to +Infinity including -0, Number.MIN_VALUE, Number.EPSILON and others (from small to large values).